x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchRequest.java

```diff
@@ -135,6 +141,13 @@ public EqlSearchRequest(StreamInput in) throws IOException {
         if (in.getTransportVersion().onOrAfter(TransportVersions.V_8_7_0)) {
             maxSamplesPerKey = in.readInt();
         }
+        if (in.getTransportVersion().onOrAfter(TransportVersions.EQL_ALLOW_PARTIAL_SEARCH_RESULTS)) {
```

Comment:
Do we have any tests for serialization to make sure in a mixed cluster tests everything is ok from this point of view?

Reply:
BWC serialization is unit tested in `EqlSearchRequestTests`.
I'm adding some randomization to mixed-node tests